protocol/avdl/keybase1/gui_file_access.avdl

```diff
@@ -0,0 +1,129 @@
+@namespace("keybase.1")
+
+protocol GuiFileAccess {
+
+  import idl "common.avdl";
+
+/*
+  File manipulation interface
+  
+  Caller provides OpID to asynchronous method call, repeats until status is no
+  longer pending or operation is cancelled.
+ 
+  KBFS paths are expected to start with "private" or "public" - mount name is implied.
```

Comment:
I assume the non-KBFS paths have to be absolute (i.e., start with `/` or a drive letter)?